docker-for-aws/load-balancer.md

```diff
@@ -0,0 +1,83 @@
+
+## How can I configure my load balancer to support SSL/TLS traffic?
+
+Yes, you can. We are using [Amazons' ACM service](https://aws.amazon.com/certificate-manager/) which provides free SSL/TLS certificates, which can be used with ELBs. You will need to create a new certificate for your domain, and get the ARN for that certificate.
+
+You will add a label to your service to tell swarm that you want to use a given ACM cert for SSL connections to your service.
+
+### Here are some examples
+
+Start a service and listen on the ELB with ports `80` and `443`. Port `443` will be served using the SSL from the ACM given in label `com.docker.aws.lb.arn`
+
+```
+docker service create --name demo --detach=true --publish 80:80 --publish 443:80 --label com.docker.aws.lb.arn="arn:aws:acm:us-east-1:0123456789:certificate/c02117b6-2b5f-4507-8115-87726f4ab963" yourname/your-image:latest
+```
+
+By default when you add an ACM ARN as a label, it will listen on port `443`. If you want to change which port to listen too you will append an `@` symbol and a list of ports you want to expose.
+
+#### links SSL to port 443
+
+```
+com.docker.aws.lb.arn="arn:..."
+```
+
+#### links SSL to port 444
+
+```
+com.docker.aws.lb.arn="arn:...@444"
+```
+
+#### links SSL to ports 444 and 8080
+
+```
+com.docker.aws.lb.arn="arn:...@444,8080"
+```
+
+### More Full examples:
+
+listen for HTTP on ports 80 and HTTPS on 444
+
+```
+docker service create --name demo --detach=true --publish 80:80 --publish 444:80 --label com.docker.aws.lb.arn="arn:aws:acm:us-east-1:0123456789:certificate/c02117b6-2b5f-4507-8115-87726f4ab963@444" yourname/your-image:latest
+```
+
+#### SSL Listen on port 444 and 443
+
+```
+docker service create --name demo --detach=true --publish 80:80 --publish 444:80 --label com.docker.aws.lb.arn="arn:aws:acm:us-east-1:0123456789:certificate/c02117b6-2b5f-4507-8115-87726f4ab963@443,444" yourname/your-image:latest
+```
+
+#### SSL Listen on port 8080
+
+```
+docker service create --name demo --detach=true --publish 8080:80 --label com.docker.aws.lb.arn="arn:aws:acm:us-east-1:0123456789:certificate/c02117b6-2b5f-4507-8115-87726f4ab963@8080" yourname/your-image:latest
+```
+
+### Add a CNAME for your ELB
+
+Once you have your ELB setup, with the correct listeners and certificates, you will need to add a DNS CNAME that points to your ELB at your DNS provider.
+
+### ELB SSL Limitations
+
+- There can only be one SSL certificate per ELB, so that means you can only have one label per swarm. If you add more than one `com.docker.aws.lb.arn` label per swarm the last one loaded will overwrite the other one.
+- If you remove the service that has the `com.docker.aws.lb.arn` label it will be removed from the ELB.
+- If you edit the ELB config directly from the dashboard, the changes will not stay, they will be whipped out after the next update.
+
+## Can I make manual configurations to the ELB?
```

Comment:
"make manual configurations" sounds weird. Do you mean "make manual configuration changes" or "manually configure" or something like that?